server/methods/getUsersOfRoom.js

```diff
@@ -44,7 +62,7 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-allowed', 'Not allowed', { method: 'getUsersOfRoom' });
 		}
 
-		const total = Subscriptions.findByRoomIdWhenUsernameExists(rid).count();
+		const total = findUsersActiveTotal({ rid });
```

Comment:
Hey! What about modifying `findUsers` to return a `cursor`, and then fetch the total & results from the same cursor? In that way, there's no need to do this additional call 😬 what do you think?

Reply:
@KevLehman Thanks for your review. I think it's a very good idea, I'm making the changes now.